src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs

```diff
@@ -2845,6 +2846,7 @@ private bool SpinThenBlockingWait(int millisecondsTimeout, CancellationToken can
                 try
                 {
                     AddCompletionAction(mres, addBeforeOthers: true);
+#pragma warning disable CA1416 // Validate platform compatibility, Need proper handling
```

Comment:
If I add [UnsupportedOSPlatform("browser")] for this API it will affect all Task.Wait(...) overloads and other APIs, suppressed for now as we might want different handling for browser